### PR TITLE
Fix text for datasets button in org page

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/792

### CODE REVIEW

File: udata_front/theme/gouvfr/templates/organization/display.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h2>{{ _('Datasets') }}<sup>{{ total_datasets }}</sup></h2>
             href="{{ url_for('datasets.list', organization=org.id) }}"
             class="fr-btn fr-btn--secondary fr-icon-eye-line fr-btn--icon-left"
         >
-            {{ _('See %(count)s datasets', count=total_datasets ) }}
+            {{ ngettext('See {0} dataset', 'See {0} datasets', total_datasets) }}
```
Comment (maudetes) [inline udata_front/theme/gouvfr/templates/organization/display.html:108]: From what I understand from ngettext, we should use `%(num)d`: https://pythonhosted.org/Flask-BabelEx/#flask.ext.babelex.Domain.ngettext

**Review by maudetes — COMMENTED** [review]
Thank you for the modification!

Don't forget to re-extract translation ;)
I'm wondering about the singular string, should it be "See the dataset" instead of "See {0} dataset" (thinking of the French translation as well!

**Comment by nicolaskempf57** [comment]
> Thank you for the modification!
> 
> Don't forget to re-extract translation ;) I'm wondering about the singular string, should it be "See the dataset" instead of "See {0} dataset" (thinking of the French translation as well!

Done !

**Review by maudetes — APPROVED** [review]
:ok_hand:

**Pull request merged**